tests/integration/targets/lookup_hashi_vault/runme.sh

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env bash
 
+# The default container does not seem to have virtualenv installed
+${ANSIBLE_TEST_PYTHON_INTERPRETER} -m pip install virtualenv
+
 set -eux
 
-ansible-galaxy -vvv collection install community.crypto
+source virtualenv.sh
```

Comment:
I did finally figure out that the issues were venv-based because of some combination of the ansible container and `ansible-test`'s use of virtual environments, but I've never heard of `virtualenv.sh`, is this documented anywhere at all??

This stuff was driving me crazy, thank you.

Reply:
I have no idea idea where/whether it is documented. I mainly know that some `runme.sh` based tests use it (and need it), to make sure that the modules are executed with the correct Python version (which is not necessarily equal to the Python version with which ansible-playbook is run).